tests/report_status_test.go

```diff
@@ -84,6 +84,15 @@ func GetPodsLogs() (string, error) {
 	return buffer.String(), nil
 }
 
+func FindInterface(name string, interfaces []nmstatev1.InterfaceInfo) (*nmstatev1.InterfaceInfo, bool) {
```

Comment:
if shouldn't be public, nobody should import *_test

Reply:
done